Intersect.Client/Entities/Entity.cs

```diff
@@ -1240,8 +1304,8 @@ public virtual float GetTop(int overrideHeight = -1)
 
         public virtual void DrawName(Color textColor, Color borderColor = null, Color backgroundColor = null)
         {
-            // Are we supposed to hide this name?
-            if (!ShouldDrawName || string.IsNullOrWhiteSpace(Name))
+            // Are we really supposed to draw this name?
+            if (!ShouldDrawName)
```

Comment:
Why remove the IsNullOrWhiteSpace check?

Reply:
because it's already happening within `ShouldDrawName`, which will be false in the case of `string.IsNullOrWhiteSpace(Name)`